handlers/stations.go

```diff
@@ -258,8 +260,19 @@ func (umh StationsHandler) CreateStation(c *gin.Context) {
 		c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
 		return
 	}
-
-	logger.Info("Station " + stationName + " has been created")
+	message := "Station " + stationName + " has been created"
+	logger.Info(message)
+	var auditLogs []interface{}
+	newAuditLog := models.AuditLog{
+		ID:              primitive.NewObjectID(),
+		StationName:     stationName,
+		Message:       	 message,
+		CreatedByUser:   user.Username,
+		CreationDate:    time.Now(),
+		UserType: 		 user.UserType,
+	}
+	auditLogs = append(auditLogs, newAuditLog)
+	CreateAuditLogs(auditLogs)
```

Comment:
this should get an error and handle it here, you can log the error with warn flag